src/screens/PodcastsScreen.tsx

```diff
@@ -1,19 +1,23 @@
 import AsyncStorage from '@react-native-community/async-storage'
-import { TouchableOpacity } from 'react-native'
+import { FlatList } from 'react-native-gesture-handler'
```

Comment:
This seems weird. Should the FlatList be coming from he gesture handler?

Reply:
Oh wow, that autoimported and I didn't realize it. Since the FlatList is going to become its own component anyway (going to work on that today), I say we keep this as is for now.